frontend/src/components/pages/CampOverview/CampersTable/CampersTable.tsx

```diff
@@ -96,6 +96,8 @@ const CampersTable = ({
     specialNeeds: 0,
   });
 
+  const [selectedCamper, setSelectedCamper] = React.useState<Camper>();
```

Comment:
If we have a null state (you check that later), should we have the type `<Camper | null>`? or `<Camper | undefined>` maybe